src/extension.ts

```diff
@@ -872,6 +872,13 @@ function launchMetals(
         });
       });
 
+      registerCommand(`metals.reset-notifications`, (args = []) => {
```

Comment:
is `= []` needed?

Reply:
Probably not :sweat_smile: but didn't want to angry the typescript gods just in case